lib/output.js

```diff
@@ -2047,10 +2077,21 @@ function OutputStream(options) {
         self.value._do_print(output, true);
     });
     DEFPRINT(AST_ObjectSetter, function(self, output) {
-        self._print_getter_setter("set", output);
+        self._print_getter_setter("set", false, output);
     });
     DEFPRINT(AST_ObjectGetter, function(self, output) {
-        self._print_getter_setter("get", output);
+        self._print_getter_setter("get", false, output);
+    });
+    DEFPRINT(AST_PrivateMethod, function(self, output) {
+        var type;
+        if (self.is_generator && self.async) {
+            type = "async*";
+        } else if (self.is_generator) {
+            type = "*";
+        } else if (self.async) {
+            type = "async";
+        }
```

Comment:
may be it would be better to write this way?

```suggestion
        var type = '';
        if (self.async) {
            type += "async";
        }
        if (self.is_generator) {
            type += "*";
        }
```

Reply:
This is taken from an exisitng similar node type: if we change the logic here we should change both; now I'm doing a drive by fix which is not ideal...